src/ReflectionParameter.php

```diff
@@ -320,6 +320,10 @@ public function getType(): ?\ReflectionType
             return null;
         }
 
+        if ($parameterType === 'iterable') {
+            $parameterType = 'Traversable|array';
+        }
```

Comment:
@lisachenko this is needed as php 8.2 display `Traversable|array` for `iterable, see https://3v4l.org/3uVi3

Reply:
Oh, nice )) Haven't been aware about this 👍